### ISSUE TITLE
qtox: declare dependance of >=dev-libs/libsodium-0.6.1

### ISSUE THREAD

**Issue by ghost** [zetok/gentoo-overlay-tox#57]
qtox uses crypto_sign_verify_detached(), lacking before 0.6.1.

Thanks.

**Comment by zetok** [comment]
```
[I] dev-libs/libsodium
     Available versions:  ~0.6.1(0/10) ~0.7.0(0/13) ~0.7.1(0/13) (~)1.0.0(0/13)
```

From what I see, there's no version lower than 0.6.1 of libsodium in Portage, and haven't been since quite some time.

Is there a reason why would that needed to be explicitly set again?

I kinda wonder whether Tox overlay could be used by other than Gentoo distros using Portage. Is that the case? And if so, then I assume your distro still offers older libsodium versions?

**Comment by ghost** [comment]
Hm. Seems 0.6.0 is really not there in portage anymore, but i believe i
have done world upgrade nearly two weeks ago. And i had libsodium-0.6.0-r1
now and it lead to qtox compilation failure.

## 

Andrey Utkin

**Comment by urras** [comment]
Hi,
Fixed in 4e03a5a. Toxcore now requires Libsodium 0.6.1 or later.
